Build fails when manpages selected

Minor issue - `/lib` in `target` seems to be missing or not automatically created.


```
install bc/bc to bin
rm Filelist
for DIR in ; do /Library/Developer/CommandLineTools/usr/bin/make -C $DIR install || exit 1; done
mkdir /Users/helge/src/mellvik/elks/target/lib/man1
mkdir: /Users/helge/src/mellvik/elks/target/lib: No such file or directory
make[3]: *** [install] Error 1
make[2]: *** [template] Error 2
make[1]: *** [image] Error 2
make: *** [all] Error 2
```

--- a/elkscmd/Make.install
+++ b/elkscmd/Make.install
@@ -213,15 +213,15 @@ ifdef CONFIG_APP_CGATEXT
 	make -C cgatext install
 endif
 ifdef CONFIG_APP_MAN_PAGES
-	mkdir $(DESTDIR)/lib/man1
+	mkdir -p $(DESTDIR)/lib/man1
 	cp -rp man/man1 $(DESTDIR)/lib; compress -b 12 $(DESTDIR)/lib/man1/*
-	mkdir $(DESTDIR)/lib/man2
+	mkdir -p $(DESTDIR)/lib/man2
 	cp -rp man/man2 $(DESTDIR)/lib; compress -b 12 $(DESTDIR)/lib/man2/*
-	mkdir $(DESTDIR)/lib/man5;
+	mkdir -p $(DESTDIR)/lib/man5
 	cp -rp man/man5 $(DESTDIR)/lib; compress -b 12 $(DESTDIR)/lib/man5/*
-	mkdir $(DESTDIR)/lib/man7;
+	mkdir -p $(DESTDIR)/lib/man7
 	cp -rp man/man7 $(DESTDIR)/lib; compress -b 12 $(DESTDIR)/lib/man7/*
-	mkdir $(DESTDIR)/lib/man8;
+	mkdir -p $(DESTDIR)/lib/man8
 	cp -rp man/man8 $(DESTDIR)/lib; compress -b 12 $(DESTDIR)/lib/man8/*
 endif
 

